PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -1174,6 +1174,9 @@ void PATElectronProducer::embedHighLevel( pat::Electron & anElectron,
   d0_corr = result.second.value();
   d0_err = beamspotIsValid ? result.second.error() : -1.0;
   anElectron.setDB( d0_corr, d0_err, pat::Electron::BS3D);
+
+    // PVDZ
+  anElectron.setDB( track->dz(primaryVertex.position()), track->dzError(), pat::Electron::PVDZ );
```

Comment:
shouldn't this include the PV z error?
All of the tip [d0] variants above have it.

Reply:
based on the PR description
> save dz(PV) into the pat Electron & Muon for convenience of applying it downstream without a PV object

it sounds like the PV error on z should be added.

Reply:
For d0 and ip3d, the b-tag tool does the right error propagation but I don't think it's implemented for dz.
I could do something simple like `std::hypot(track->dzError(), primaryVertex.zError())` , even though I don't think it's really the right math.

Reply:
it seems good enough. It's used in a few places (e.g.  CommonTools/ParticleFlow/interface/IPCutPFCandidateSelectorDefinition.h, Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc, DQM/TrackingMonitor/src/TrackAnalyzer.cc, and at least as many more, based on git grep)

perhaps a more complete would be to deconstruct the Track::dz method and propagate all relevant uncertainties, (to be added to TrackBase? or to the IPTools)

Reply:
pushed the simple hypot version